libs/counterfactuals/src/lib/CounterfactualList.styles.ts

```diff
@@ -32,7 +32,7 @@ export const counterfactualListStyle: () => IProcessedStyleSet<ICounterfactualLi
         }
       },
       editCell: {
-        color: theme.palette.green,
+        color: theme.semanticColors.bodyText,
```

Comment:
is there some more interesting color we could use here?  or is it really recommended to just use regular body text for accessibility?

Reply:
color is useless for people with vision issue, suggest in future if we need to convey different information, we can use other signals instead of color, in this PR's table it only indicates a difference between the first row and the rest of the information. So just change it to black per suggestion from designer.